community-operators/postgresql/postgres-operator.v4.0.1.clusterserviceversion.yaml

```diff
@@ -156,47 +294,22 @@ spec:
     name: Crunchy Data
   labels:
     alm-owner-postgres: postgres-operator
-    alm-status-descriptors: postgres-operator.v3.5.0
+    alm-status-descriptors: postgres-operator.v4.0.1
   links:
   - name: Crunchy Data
     url: https://www.crunchydata.com
   - name: Documentation
-    url: https://crunchydata.github.io/postgres-operator/3.5.0/
+    url: https://access.crunchydata.com/documentation/postgres-operator/4.0.1/
   selector:
     matchLabels:
       alm-owner-postgres: postgres-operator
   customresourcedefinitions:
     owned:
-    - name: pgbackups.cr.client-go.k8s.io
-      version: v1
-      kind: Pgbackup
-      displayName: Pgbackup
-      description: An instance of Pgbackup.
     - name: pgclusters.cr.client-go.k8s.io
```

Comment:
Change this name to pgclusters.crunchydata.com. Also apply the CRD group name change here: https://github.com/operator-framework/community-operators/blob/44fdd6681e974ffce5910e54024e44ccc6c6037b/upstream-community-operators/postgresql/pgcluster.crd.yaml#L4 and here: https://github.com/operator-framework/community-operators/blob/44fdd6681e974ffce5910e54024e44ccc6c6037b/upstream-community-operators/postgresql/pgcluster.crd.yaml#L6

Reply:
yeah, that was an oversight - those will get updated shortly.